pynitrokey/cli/__init__.py

```diff
@@ -58,10 +61,30 @@ def nitropy():
     handler = logging.FileHandler(filename=LOG_FN, delay=True, encoding="utf-8")
     logging.basicConfig(format=LOG_FORMAT, level=logging.DEBUG, handlers=[handler])
 
+    # Timestamp for log
+    logger.info(f"Timestamp: {datetime.now()}\n")
+    # Info Os
+    logger.info(f"OS: {platform.uname()}\n")
+    # Info Python version
+    logger.info(f"Python version: {platform.python_version()}\n")
```

Comment:
Please remove the `\n` chars, and all the comments (the data meaning is pretty self-describing, no need for them)
```suggestion
    logger.info(f"Timestamp: {datetime.now()}")
    logger.info(f"OS: {platform.uname()}")
    logger.info(f"Python version: {platform.python_version()}")
```

Reply:
got it